src/app/profile/view.nim

```diff
@@ -13,6 +15,8 @@ QtObject:
     profile*: ProfileInfoView
     mailserversList*: MailServersList
     contactList*: ContactList
+    mnemonicLeft*: MnemonicList
+    mnemonicRight*: MnemonicList
```

Comment:
Managing two `AbstractListModels` here because I'm not sure if we can have a single `ListView` break split into two columns automatically in QML

Reply:
IMO, handling the mnemonic like a list is  a bit overkill. Since it always has the same syntax of `word space word`, we could just pass to the QML the mnemonic as a string, like we handle in most places, and let the QML handle it. Since it has JS support, it's going to be easy for it to split it and loop over it. Using a ListView is also overkill since we could just show each word as an `Item` and handle the columns like that (if `index` > `length/2`, anchor to the right)